### PR TITLE
Aaa/level update

### PR DESCRIPTION
**Pull request by jchapmanio**
A whole new set of tile prefabs that improve the look and feel of the FPS world.
Tiles now take advantage of nested prefabbing.
Removed the centre tiles logic from mapbuilder.cs

Modified the spawn point indicators to be more visible + easier to select.

**Not ready for merging**. There is still some cosmetic content incoming from our art dept. I am making this PR to give some visibility to what we'll be changing and so we can catch any potential problems earlier.

### CODE REVIEW

**Comment by jchapmanio** [comment]
I have built navmesh and tested on mobile and works acceptably. Difficult to compare performance directly with previous tiles, but nothing seems obviously wrong.

**Comment by paulbalaji** [comment]
can you rebase off the latest `develop` and change your branch name to `feature/aaa-level-update`? This is so you get the buildkite PR that has gone in + TeamCity can pick up this branch (it doesn't pick up branches with the `aaa/` prefix)

File: workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/TileFootprintHelper.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+using UnityEditor;
+using UnityEditor.Experimental.SceneManagement;
+
+#endif
+
+
+public class TileFootprintHelper : MonoBehaviour
+{
+#if UNITY_EDITOR
+    public float TileSize = 8 * 4;
```
Comment (paulbalaji) [inline workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/TileFootprintHelper.cs]: have the tile sizes changed now? I thought they were 36x36

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,6 +29,10 @@
 - Increased the level size from 868x868 to 1732x1732.
 - Changed how level generation and `MapBuilder.cs` works.
 	- `LoadWorld()` now generates the level GameObject through `MapBuilder`, instead of instantiating it from a prefab.
+	- The four middle tiles of the world are no longer forced to be of certain tile type.
```
Comment (paulbalaji) [inline CHANGELOG.md]: this entry will need to go into the latest `Changed` section, after rebasing

File: workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/TileFootprintHelper.cs (hunk 1 above)
Comment (jchapmanio) [reply workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/TileFootprintHelper.cs]: the full footprint of the tile is 36m (8 * 4m 'squares' + a 4m gutter square), this helper is the inner part of the tile, the 8 * 4m part.

Comment (jchapmanio) [reply workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/TileFootprintHelper.cs]: In the new branch, I've renamed the var to `InnerTilerSize` to make it clearer what it represents

**Comment by jchapmanio** [comment]
I have made a new branch, `feature/aaa-level-update`, rebased and fixed some issues. Shall I do a separate PR, or is there a way to pull it into this PR?

**Comment by jchapmanio** [comment]
Closing PR as `feature/aaa-level-update` now exists.

**Pull request closed without merging**